src/pages/Landing/__snapshots__/index.test.tsx.snap

```diff
@@ -2027,25 +2027,25 @@ exports[`disable nft on landing page does not render nft information and card 1`
         </div>
       </div>
       <span
-        class="1 c55"
+        class="c55"
       >
         <a
           class="c2 c25 c56 c57"
           href="#/swap"
         >
           <p
-            class="0 c58"
+            class="c58"
           >
             Get started
           </p>
         </a>
       </span>
       <div
-        class="2 c59"
```

Comment:
🤨 what were these

Reply:
vanilla-extract is now omitted from unit tests, so the extra classes are removed from snapshots.